src/taskgraph/run-task/run-task

```diff
@@ -570,6 +571,7 @@ def _clean_git_checkout(destination_path):
         cwd=destination_path,
         env=os.environ,
     )
+    _display_python_version_in_sub_process(subprocess)
```

Comment:
@jcristau, correct me if I'm wrong: the python binaries this script has access to cannot change during the execution, right? If so, we don't need to display the version of python every time we shell out, do we?